### ISSUE TITLE
Issue in readme - frcnn.py: error: unrecognized arguments: --lr_decay_step=250000

### ISSUE THREAD

**Issue by saiabinesh** [kshmelkov/incremental_detectors#5]
Full error Below

usage: frcnn.py [-h] --run_name RUN_NAME [--ckpt CKPT]
                [--dataset {voc07,voc12,coco}] [--proposals {mcg,edgeboxes,}]
                --num_classes NUM_CLASSES [--extend EXTEND]
                [--num_layers NUM_LAYERS] --action ACTION
                [--data_format {NHWC,NCHW}] [--sigmoid]
                [--print_step PRINT_STEP] [--conf_thresh CONF_THRESH]
                [--nms_thresh NMS_THRESH] [--eval_first_n EVAL_FIRST_N]
                [--eval_ckpts EVAL_CKPTS] [--batch_size BATCH_SIZE]
                [--num_images NUM_IMAGES]
                [--num_positives_in_batch NUM_POSITIVES_IN_BATCH]
                [--pretrained_net PRETRAINED_NET] [--train_vars TRAIN_VARS]
                [--optimizer {adam,nesterov,sgd,momentum}]
                [--weight_decay WEIGHT_DECAY] [--learning_rate LEARNING_RATE]
                [--lr_decay LR_DECAY [LR_DECAY ...]]
                [--max_iterations MAX_ITERATIONS] [--reset_slots RESET_SLOTS]
                [--frcnn_loss_coef FRCNN_LOSS_COEF]
                [--class_distillation_loss_coef CLASS_DISTILLATION_LOSS_COEF]
                [--bbox_distillation_loss_coef BBOX_DISTILLATION_LOSS_COEF]
                [--distillation] [--bias_distillation] [--crossentropy]
                [--smooth_bbox_distillation]
                [--num_dataset_readers NUM_DATASET_READERS]
                [--num_prep_threads NUM_PREP_THREADS] [--filter_proposals]
                [--prefetch_all]
frcnn.py: error: unrecognized arguments: --lr_decay_step=250000
